tests/integration/test_dbfs_store/test_dbfs_store.py

```diff
@@ -0,0 +1,216 @@
+
+            response = data_item.get(offset=20)
+            assert response.decode() == test_string[20:]
+
+        finally:
+            if use_secrets_as_parameters:
+                os.environ["DATABRICKS_TOKEN"] = token
+
+    def test_stat(self):
+        data_item, _ = self._get_data_item()
+        data_item.put(test_string)
+        stat = data_item.stat()
+        assert stat.size == len(test_string)
+
+    def test_list_dir(self):
+        data_item, object_url = self._get_data_item()
+        data_item.put(test_string)
+        dir_dataitem = mlrun.run.get_dataitem(
+            self._dbfs_url + self.test_root_dir,
+        )
+        dir_list = dir_dataitem.listdir()
+        assert object_url.split("/")[-1] in dir_list
+
+    def test_upload(self):
+        data_item, _ = self._get_data_item()
+        data_item.upload(test_file_path)
+        response = data_item.get()
+        assert response.decode() == test_string
+
+    def test_rm(self):
+        data_item, _ = self._get_data_item()
+        data_item.upload(test_file_path)
+        data_item.stat()
+        data_item.delete()
+        with pytest.raises(FileNotFoundError) as file_not_found_error:
+            data_item.stat()
+        assert "No file or directory exists on path" in str(file_not_found_error.value)
+
+    @pytest.mark.parametrize(
+        "file_extension, local_file_path, reader",
+        [
+            (
+                "parquet",
+                parquet_path,
+                pd.read_parquet,
+            ),
+            ("csv", csv_path, pd.read_csv),
+            ("json", json_path, pd.read_json),
+        ],
+    )
+    def test_as_df(self, file_extension: str, local_file_path: str, reader: callable):
+        source = reader(local_file_path)
+        upload_file_path = (
+            f"{self.test_root_dir}/file_{str(uuid.uuid4())}.{file_extension}"
+        )
+        upload_data_item = mlrun.run.get_dataitem(
+            self._dbfs_url + upload_file_path,
+        )
+        upload_data_item.upload(local_file_path)
+        response = upload_data_item.as_df()
+        assert source.equals(response)
+
+    @pytest.mark.parametrize(
+        "directory, file_format, file_extension, files_paths, reader",
+        [
+            (
+                PARQUETS_DIR,
+                "parquet",
+                "parquet",
+                [parquet_path, additional_parquet_path],
+                pd.read_parquet,
+            ),
+            (CSV_DIR, "csv", "csv", [csv_path, additional_csv_path], pd.read_csv),
+        ],
+    )
+    def test_check_read_df_dir(
+        self,
+        directory: str,
+        file_format: str,
+        file_extension: str,
+        files_paths: List[Path],
+        reader: callable,
+    ):
+        first_file_path = files_paths[0]
+        second_file_path = files_paths[1]
+        uploaded_file_path = (
+            f"{self.test_root_dir}{directory}/file_{str(uuid.uuid4())}.{file_extension}"
```

Comment:
`str` in an f-string is redundant (elsewhere as well).
```suggestion
            f"{self.test_root_dir}{directory}/file_{uuid.uuid4()}.{file_extension}"
```